great_expectations/datasource/fluent/config.py

```diff
@@ -222,7 +222,7 @@ def _load_datasource_subtype(cls, v: List[dict]):
                 for asset in datasource.assets:
                     asset._datasource = datasource
 
-        logger.info(f"Loaded 'datasources' ->\n{repr(loaded_datasources)}")
+        logger.debug(f"Loaded 'datasources' ->\n{repr(loaded_datasources)}")
```

Comment:
This is way too verbose in our test logs when logging more than 1 or 2 datasources.